requirements/common.txt

```diff
@@ -2,3 +2,5 @@ Django==1.9.3
 pyaml==15.8.2
 django-allauth==0.30.0
 psycopg2==2.7.3
+scikit-image==0.13.1
+matplotlib==2.1.0
```

Comment:
Aren't these already installed in `caffe_tensorflow_keras_install.sh`?

Reply:
Matplotlib isnt in that script. scikit-image does not get installed through the script, idk why, but will remove it if you insist.

Reply:
The travis build was fine before as well so `scikit-image` must have been getting installed through the script, and `matplotlib` is not there separately as its a dependency for `scikit-image`  and is installed automatically.

Reply:
I tried installing twice on different VMs, and faced the issues. Even @manuhortet faced this issue. (Need for installing scikit-image and matplotlib)

Anyways, removed the two lines.